_std/pathfinding.dm

```diff
@@ -446,7 +452,7 @@
 						continue
 					else
 						return FALSE
-				return FALSE
+				return A.Cross(passer)
```

Comment:
Using return here will stop checking objects if one is found to be crossable, see the weird conditional return below